ale/config.py

```diff
@@ -7,3 +7,4 @@
 cassini = '/data/big/spice/co-s_j_e_v-spice-6-v1.0/cosp_1000/extras/mk'
 mdis = '/data/spice/mess-e_v_h-spice-6-v1.0/messsp_1000/extras/mk' # Messenger
 mro = '/data/spice/mro-m-spice-6-v1.0/mrosp_1000/extras/mk' # Mars Reconnaissance Orbiter
+kaguya = '/data/fast/kernels/SELENE/kernels/mk'
```

Comment:
I'm not sure if a closed PR is an okay place to ask this, but, where is this `/data/` located? I can't see it from the prog systems.